### ISSUE TITLE
Application to unseen data

### ISSUE THREAD

**Issue by edrdos101** [layumi/Person_reID_baseline_pytorch#106]
Hi! Thanks a lot for the tutorial, I found it helped me a lot in understanding the reID concepts.

I was wondering if you could advise how the model trained on Market data can be applied to unseen data - from a completely different setting. I guess in my limited understanding the typical process is - train a model on 70% of dataset A, test on 30% of dataset A; if it performs well then try on dataset B,C,D etc... Now my question is, would this be possible here?

As a follow on, how do you then use the features to id and then re-id people? For example Deep_SORT repo (https://github.com/nwojke/deep_sort) has a demo which identifies people and the re-ids them on a video. I had much better time understanding how to train models with your code, so would like to use it for labelling some of my own data and perhaps get a nice visual demo going

Would really appreciate any pointers!

**Comment by layumi** [comment]
Hi @edrdos101 ,

Thank you. I think you may use the Market test set as validation set to tune one model. Then you may apply it to other datasets.

Actually, there are already some works about it. You may refer to this table (They all train the model on Market-1501 and then apply the model to an unseen dataset DukeMTMC-reID). The link is [here](https://github.com/layumi/DukeMTMC-reID_evaluation/tree/master/State-of-the-art#transfer-learning).
